packages/client/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -1,45 +1,80 @@
-import React from 'react';
-import cn from 'classnames';
-import Input from '../../components/Input';
-import Button from '../../components/Button';
-import Form from '../../components/Form';
-import styles from './LoginPage.module.scss';
+import React from 'react'
+import { Link } from 'react-router-dom'
+import cn from 'classnames'
+import { useFormik } from 'formik'
+import Input from '../../components/Input'
+import Button from '../../components/Button'
+import Form from '../../components/Form'
+import styles from './LoginPage.module.scss'
+
+import { loginSchema } from '../../constants/Schemas'
 
 const LoginPage: React.FC = (): JSX.Element => {
+  const { values, errors, touched, handleChange, handleSubmit, handleBlur } =
+    useFormik({
+      initialValues: {
+        login: '',
+        password: '',
+      },
+      validationSchema: loginSchema,
+      onSubmit: values => {
+        console.log('values', values)
+      },
+    })
+
   return (
-    <div>
+    <div className={styles.form_root}>
       <h1 className={styles.form_logo_title}>
-        {'Huggy Wuggy'}
-        <br />
-        {'& Kissy Missy'}
+        Huggy Wuggy
+        <br />& Kissy Missy
       </h1>
       <Form
+        onSubmit={handleSubmit}
         actions={[
-          <div className={styles.form_buttons}>
+          <div key={0}>
             <div className={styles.form_button_box}>
-              <Button>
-                Sign in
+              <Button className={styles.form_button} type="submit">
```

Comment:
```suggestion
              <Button className={styles.form_button} type="submit">
```

              <Button color="regular" type="submit">
                <h1 className={styles.login_button_title}>Sign in</h1>
              </Button>